Lib/test/test_interpreters.py

```diff
@@ -463,6 +464,27 @@ def test_bytes_for_script(self):
     # test_xxsubinterpreters covers the remaining Interpreter.run() behavior.
 
 
+class StressTests(TestBase):
+
+    # In these tests we generally want a lot of interpreters,
+    # but not so many that any test takes too long.
+
+    def test_create_many_sequential(self):
```

Comment:
```suggestion
    @requires_resource('cpu')
    def test_create_many_sequential(self):
```

Reply:
This test isn't more CPU-intensive than any other.  What's the value of the decorator here?

Reply:
Okay let's skip it. :)